csrc/ir/utils.h

```diff
@@ -58,7 +58,9 @@ MmaOpDetails getMmaOpDetails(
     TensorView* in_a,
     TensorView* in_b);
 
-void verifyMmaOpForEvaluation(MmaOp* mma_op, DataType castop_out_dtype);
+void verifyMmaOpForEvaluation(
+    MmaOp* mma_op,
+    const DataType expected_input_dtype);
```

Comment:
```suggestion
    DataType expected_input_dtype);
```

https://abseil.io/tips/109